code/modules/power/portgen.dm

```diff
@@ -355,13 +358,13 @@
 
 	return data
 
-/obj/machinery/power/port_gen/pacman/ui_interact(mob/user)
+/obj/machinery/power/portgen/basic/ui_interact(mob/user)
 	var/datum/vueui/ui = SSvueui.get_open_ui(user, src)
 	if (!ui)
 		ui = new(user, src, "machinery-power-pacman", 500, 560, capitalize(name))
```

Comment:
```suggestion
		ui = new(user, src, "machinery-power-portgen", 500, 560, capitalize(name))
```
i want to change it to this, but doing so doesn't work. i can't find this anywhere else in the code.